cypress/integration/cucumber/step_definitions/refapp-3.x/01-login/login.js

```diff
@@ -0,0 +1,24 @@
+import {Given} from 'cypress-cucumber-preprocessor/steps';
+
+Given('user arrives at the login page', () => {
+    cy.task('getProperty', 'webapp.url').then(baseUrl => {
```

Comment:
am wondering if we should extract out a re-usable function/command for visiting a url. otherwise this looks nice

Reply:
Yeah that would be nice! I'll do it!

Reply:
Fixed.